### ISSUE TITLE
Clocker.v22.02.01 was 'universal', Clocker.v22.05 isn't

### ISSUE THREAD

**Issue by core-code** [n0shake/clocker#107]
```
file Desktop/Clocker.v22.02.01/Clocker.app/Contents/MacOS/Clocker
Desktop/Clocker.v22.02.01/Clocker.app/Contents/MacOS/Clocker: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit executable x86_64
- Mach-O 64-bit executable x86_64] [arm64]
Desktop/Clocker.v22.02.01/Clocker.app/Contents/MacOS/Clocker (for architecture x86_64):	Mach-O 64-bit executable x86_64
Desktop/Clocker.v22.02.01/Clocker.app/Contents/MacOS/Clocker (for architecture arm64):	Mach-O 64-bit executable arm64

```
```
file Desktop/Clocker.v22.05/Clocker.app/Contents/MacOS/Clocker
Desktop/Clocker.v22.05/Clocker.app/Contents/MacOS/Clocker:    Mach-O 64-bit executable x86_64
```

**Comment by n0shake** [comment]
This is a KP; and will be mitigated in the next update!

**Comment by n0shake** [comment]
Fixed in v22.06!

**Comment by core-code** [comment]
awesome, thanks!
